src/Centreon/Infrastructure/Contact/ContactRepositoryRDB.php

```diff
@@ -507,4 +516,25 @@ protected function translateDbName(string $request): string
             $request
         );
     }
+
+    /**
+     * Get Parent Topology Name.
+     *
+     * @param integer $parentTopologyPage
+     * @return string|null
+     */
+    private function getParentTopologyName(int $parentTopologyPage): ?string
```

Comment:
this method seems not related to parent
```suggestion
    private function getTopologyNameByPage(int $parentTopologyPage): ?string
```